### DISCUSSION TITLE
Any Android client that supports OIDC?

### DISCUSSION THREAD

**Discussion by jorti** [FreshRSS/FreshRSS#6545] [Q&A] [answered]
Hi,

Anyone knows if there's any Android client that supports connecting to FreshRSS with OIDC authentication enabled? So far all I have tried fail to login.

Thanks.

**Comment by Alkarex** [comment, marked as answer]
Make sure you have set up your API password in your FreshRSS profile. Note that the `/api/` path is not supposed to have additional authentication such as OIDC, as each API has its own authentication requirements / protocol built-in.
Adding OIDC to our GReader API would be possible but would require additional work (PR welcome), and I am not aware of any client supporting it (but maybe - otherwise some of the open source ones may welcome a PR).

**Reply by jorti** [reply]
I missed that! after setting the API password it works.
Thank you.
